src/status_im/chat/screen.cljs

```diff
@@ -44,7 +45,7 @@
       [toolbar/actions [{:icon      :icons/options
                          :icon-opts {:color :black}
                          :handler   #(on-options chat-id name group-chat public?)}]]]
-     [add-contact-bar]]))
+     (when-not public? [add-contact-bar public?])]))
```

Comment:
Parameter could be `false` then for more clarity?

Reply:
wait, that's not meant to be there, sorry

Reply:
good spot